### DISCUSSION TITLE
Lifesaving Feature - AutoDisabler and Error Message

### DISCUSSION THREAD

**Discussion by valeriejmb** [valinet/ExplorerPatcher#3969] [Show and tell]
Hi, first, thank you for ExplorerPatcher. I was shocked when I bought a Windows 11 computer 2 years ago to discover that I could no longer use QuickLaunch. Microsoft has gone soft in the head if they think their UI changes were an improvement.

I also want to thank you for the feature that automatically disables ExplorerPatcher if Explorer won't open after 3 times in 10 seconds, and then puts up a message explaining that the problem might be a mismatch between EP and some Windows update. 
That was exactly my issue. I wasn't smart enough to uninstall EP right then - instead I rebooted which was a bad idea - but once I uninstalled the Windows update I was able to get back to Explorer and upgrade to the latest EP. 

Thank you for a VERY necessary tool for working in Windows. I can't live without QuickLaunch!

**Comment by CommandDroid** [comment]
QuickLaunch was what brought me to ExplorerPatcher and I do not regret it one bit! I would be very stuck without Quicklaunch

**Comment by valeriejmb** [comment]
The only thing I don't understand is why that only happened once. I rebooted so many times landing where Explorer wouldn't open. Only once did I get the behaviour I mentioned above. Why not every time?
